app/formatters.ts

```diff
@@ -112,10 +113,26 @@ const dateFormatterFromDimension = (
   return formatDateAuto;
 };
 
+type Formatter = (x: string) => string;
+
 const formatIdentity = (x: string | Date | null) => {
   return x !== DIMENSION_VALUE_UNDEFINED ? `${x}` : "–";
 };
 
+const decimalFormatter = (dim: NumericalMeasure, formatNumber: Formatter) => {
+  const res = dim.resolution;
+  const hasResolution = typeof res === "number";
+  const expFormatter = format(`${hasResolution ? `.${res}` : ""}~e`);
+  return (v: string) => {
+    if (v === "0" || v === "0.0") {
+      return formatNumber(v);
```

Comment:
In general, I am not a fan of displaying scientific notation when `e = 0`. Do you think there could be another rule introduced, to check if the number of leading zeros equals to 0 (and then apply just regular `formatNumber`?

So then we would be able to avoid cases like below, where e+0 doesn't add any information and just pollutes the UI?

<img width="715" alt="Screenshot 2022-10-24 at 09 42 03" src="https://user-images.githubusercontent.com/52032047/197473350-45a44a99-e2af-483d-8749-c143f830d5c3.png">